### PR TITLE
Frontend adjustments for Stripe refactoring

### PR DESCRIPTION
**Pull request by MrFlashAccount**
#### Tl;dr 
Closes: enso-org/cloud-v2#1185
This PR changes endpoints according to the comment left by @PabloBuchu: https://github.com/enso-org/cloud-v2/issues/1185#issuecomment-2114606101

#### This Change:
Changes endpoint names and fixes subscription flow

---

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.

### CODE REVIEW

**Review by somebody1234 — APPROVED** [review]

**Comment by PabloBuchu** [comment]
Why I am having this in the console? Is it related to the PR?

Screenshot 2024-05-27 at 14 48 29

**Comment by MrFlashAccount** [comment]
Not sure it's related to the PR, I guess it's the CORS policy in the IDE. We need to adjust them to allow additional hosts: https://github.com/enso-org/enso/blob/d4ef04bc524f2866be2f77884d82a9d5d76cc1d4/app/ide-desktop/lib/client/src/security.ts#L96

**Comment by PabloBuchu** [comment]
Small one but can we center the button?

Screenshot 2024-05-27 at 15 10 18

**Review by PabloBuchu — APPROVED** [review]

**Comment by MrFlashAccount** [comment]
> Small one but can we center the button?

It's not small :) Fixed it

**Comment by PabloBuchu** [comment]
Found one problematic bug. When I typed organization name already present in the system (names are globally unique) the modal is unresponsive. I can't change the input and resubmit nor close it

https://github.com/enso-org/enso/assets/2855109/97b8c41a-0a8d-4b17-b73f-de85a8cbbae5

**Review by PabloBuchu — CHANGES_REQUESTED** [review]
see comment with bug

**Comment by MrFlashAccount** [comment]
Oh this is wild... will take a look

**Comment by MrFlashAccount** [comment]
also, the error is stuck to the input above.

**Comment by MrFlashAccount** [comment]
@PabloBuchu fixed

File: app/gui2/index.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
   <body>
     <div id="enso-dashboard" class="enso-dashboard"></div>
     <div id="enso-chat" class="enso-chat"></div>
+    <div id="enso-portal-root" class="enso-portal-root"></div>
```
Comment (MrFlashAccount) [inline app/gui2/index.html:20]: We want to be sure that `.enso-portal-root` goes after the dashboard to stay on top of the whole app, so we put this element explicitly after the `dashboard` and `chat`

**Review by Frizi — APPROVED** [review]
Approving the index change in gui.

**Review by PabloBuchu — APPROVED** [review]
